docs/core/tools/dotnet-new.md

```diff
@@ -117,3 +120,9 @@ Create a new xUnit application targeting .NET Core 1.1:
 List all templates available for MVC:
 
 `dotnet new mvc -l`
+
+## See also
```

Comment:
Please add a link to the samples repo at https://github.com/dotnet/dotnet-template-samples/